package.json

```diff
@@ -38,9 +38,10 @@
     "@commitlint/config-angular": "8.2.0",
     "@commitlint/prompt": "8.2.0",
     "@commitlint/prompt-cli": "8.2.0",
-    "@edx/frontend-build": "^2.0.4",
-    "@edx/frontend-platform": "1.1.6",
-    "@edx/paragon": "7.1.5",
+    "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
+    "@edx/frontend-build": "5.4.0",
+    "@edx/frontend-platform": "1.8.0",
```

Comment:
The `peerDependency` for `@edx/frontend-platform` should be bumped to ^1.8.0 since this version is what adds `LOGO_TRADEMARK_URL` to its common config.